src/sql/src/plan/statement/ddl.rs

```diff
@@ -2201,17 +2202,28 @@ pub fn plan_create_sink(
         Some(Envelope::CdcV2) => bail_unsupported!("CDCv2 sinks"),
         Some(Envelope::None) => bail_unsupported!("\"ENVELOPE NONE\" sinks"),
     };
-    let name = scx.allocate_qualified_name(normalize::unresolved_item_name(name)?)?;
-
-    // Check for an object in the catalog with this same name
-    let full_name = scx.catalog.resolve_full_name(&name);
-    let partial_name = PartialItemName::from(full_name.clone());
-    if let (false, Ok(item)) = (if_not_exists, scx.catalog.resolve_item(&partial_name)) {
-        return Err(PlanError::ItemAlreadyExists {
-            name: full_name.to_string(),
-            item_type: item.item_type(),
-        });
+    if !is_explain && name_opt.is_none() {
+        sql_bail!("CREATE SINK requires a name!");
```

Comment:
We prefer to not use `sql_bail!` and instead use an existing `PlanError` variant or create a new one.

Reply:
I will create a new PlanError

Reply:
done!